tests/functional/deployment/test_operator.py

```diff
@@ -21,3 +24,26 @@ def test_unnecessary_pods(self):
         pods deployed.
         """
         check_unnecessary_pods_present()
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-6866")
+    def test_no_errors_in_operator_pod_logs(self, operator_pods):
+        """
+        1. Get list of all operator pods.
+        2. Check that there is no error in any of the logs.
+        """
+        pods_logs = {}
+        for operator_pod in operator_pods:
+            pod_logs = get_pod_logs(
+                pod_name=operator_pod,
+                namespace=config.ENV_DATA["cluster_namespace"],
+                all_containers=True,
+            )
+            pods_logs[operator_pod] = pod_logs
+        for operator_pod in operator_pods:
```

Comment:
As we discussed in one of the meeting with Daniel Osypenko to update some commands to be less verbose to debug output and better for memory, I was thinking about to use some simple bash command something like this:
```shell
oc logs -n openshift-storage <operator-pod> | sed 's/error: <nil>,//g' | grep -i error
```